### ISSUE TITLE
icloud mail loading issues

### ISSUE THREAD

**Issue by jmhertlein** [jmhertlein/comfydns#43]
```
Aug 03 16:13:46 claudius 0118ce86b176[40937]: 2023-08-03T16:13:46.540-0500 INFO InitialCheckingState - [Q] [192.168.1.169]: QNAME: 200608._domainkey.e.atlassian.com, QTYPE: TXT, QCLASS: IN | 5eacc677-3358-44d9-a8dc-521f51954161
Aug 03 16:13:46 claudius 0118ce86b176[40937]: 2023-08-03T16:13:46.596-0500 INFO Request - [R] [Optional[/192.168.1.169]]: SERVER_FAILURE | 5eacc677-3358-44d9-a8dc-521f51954161

```

**Comment by jmhertlein** [comment]
```
Aug 03 16:15:50 claudius 0118ce86b176[40937]: 2023-08-03T16:15:50.724-0500 INFO InitialCheckingState - [Q] [192.168.1.229]: QNAME: 200608._domainkey.email.ancestry.com, QTYPE: TXT, QCLASS: IN | b43d4987-caca-4682-97c1-8f89af1be935
Aug 03 16:15:50 claudius 0118ce86b176[40937]: 2023-08-03T16:15:50.845-0500 INFO Request - [R] [Optional[/192.168.1.229]]: SERVER_FAILURE | b43d4987-caca-4682-97c1-8f89af1be935
```

**Comment by jmhertlein** [comment]
```
Aug 03 16:16:16 claudius 0118ce86b176[40937]: 2023-08-03T16:16:16.320-0500 INFO InitialCheckingState - [Q] [192.168.1.229]: QNAME: 200608._domainkey.email.schwab.com, QTYPE: TXT, QCLASS: IN | 30e9eda4-a422-47f1-8ee2-e8c664789ac2
Aug 03 16:16:16 claudius 0118ce86b176[40937]: 2023-08-03T16:16:16.450-0500 INFO Request - [R] [Optional[/192.168.1.229]]: SERVER_FAILURE | 30e9eda4-a422-47f1-8ee2-e8c664789ac2

```

**Comment by jmhertlein** [comment]
```
Aug 03 16:16:31 claudius 0118ce86b176[40937]: 2023-08-03T16:16:31.473-0500 INFO InitialCheckingState - [Q] [192.168.1.229]: QNAME: 200608._domainkey.email.pokemon.com, QTYPE: TXT, QCLASS: IN | c31563fd-dbc7-47be-8b72-2d509be0d191
Aug 03 16:16:31 claudius 0118ce86b176[40937]: 2023-08-03T16:16:31.538-0500 INFO Request - [R] [Optional[/192.168.1.229]]: SERVER_FAILURE | c31563fd-dbc7-47be-8b72-2d509be0d191
```

**Comment by jmhertlein** [comment]
closed by cb649f81bcc62223ea506dc96d576b860281f8ff
